### PR TITLE
P16d gliders

### PR DESCRIPTION
**Pull request by cdobs**
Creates telemetered ingestions for gliders deployed on P16d

### CODE REVIEW

File: CP05MOAS-GL339/CP05MOAS-GL339_D00014_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL339/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL339-05-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-GL339/CP05MOAS-GL339_D00014_ingest.csv:6]: Looks good.

File: CP05MOAS-GL376/CP05MOAS-GL376_D00014_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL376-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL376-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL376-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL376-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL376/D00014/merged-from-glider/cp_*.mrg,CP05MOAS-GL376-05-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-GL376/CP05MOAS-GL376_D00014_ingest.csv:6]: Looks good.

File: CP05MOAS-GL379/CP05MOAS-GL379_D00013_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL379/D00013/merged-from-glider/cp_*.mrg,CP05MOAS-GL379-05-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-GL379/CP05MOAS-GL379_D00013_ingest.csv:6]: Looks good.

File: CP05MOAS-GL559/CP05MOAS-GL559_D00003_ingest.csv [hunk 4]
```diff
@@ -0,0 +1,6 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00003/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-00-ENG000000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00003/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-02-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00003/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-03-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00003/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-04-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-GL559/D00003/merged-from-glider/cp_*.mrg,CP05MOAS-GL559-05-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-GL559/CP05MOAS-GL559_D00003_ingest.csv:6]: Looks good.

File: CP05MOAS-PG564/CP05MOAS-PG564_D00004_ingest.csv [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG564/D00004/merged-from-glider/cp_*.mrg,CP05MOAS-PG564-01-CTDGVM000,telemetered,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG564/D00004/merged-from-glider/cp_*.mrg,CP05MOAS-PG564-02-DOSTAM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG564/D00004/merged-from-glider/cp_*.mrg,CP05MOAS-PG564-03-FLORTM000,telemetered,Available,
+mi.dataset.driver.moas.gl.flort_o.flort_o_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG564/D00004/merged-from-glider/cp_*.mrg,CP05MOAS-PG564-04-FLORTO000,telemetered,Available,
+mi.dataset.driver.nutnr_m.glider.nutnr_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG564/D00004/merged-from-glider/cp_*.mrg,CP05MOAS-PG564-05-NUTNRM000,telemetered,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_telemetered_driver,/omc_data/whoi/OMC/CP05MOAS-PG564/D00004/merged-from-glider/cp_*.mrg,CP05MOAS-PG564-06-PARADM000,telemetered,Available,
```
Comment (snwhite) [inline CP05MOAS-PG564/CP05MOAS-PG564_D00004_ingest.csv:8]: Looks good.

**Review by snwhite — APPROVED** [review]

File: CP05MOAS-GL339/CP05MOAS-GL339_D00014_ingest.csv (hunk 1 above)
Comment (rtravis-whoi) [reply CP05MOAS-GL339/CP05MOAS-GL339_D00014_ingest.csv:6]: good

File: CP05MOAS-GL376/CP05MOAS-GL376_D00014_ingest.csv (hunk 2 above)
Comment (rtravis-whoi) [reply CP05MOAS-GL376/CP05MOAS-GL376_D00014_ingest.csv:6]: good

File: CP05MOAS-GL379/CP05MOAS-GL379_D00013_ingest.csv (hunk 3 above)
Comment (rtravis-whoi) [reply CP05MOAS-GL379/CP05MOAS-GL379_D00013_ingest.csv:6]: good

File: CP05MOAS-GL559/CP05MOAS-GL559_D00003_ingest.csv (hunk 4 above)
Comment (rtravis-whoi) [reply CP05MOAS-GL559/CP05MOAS-GL559_D00003_ingest.csv:6]: good

File: CP05MOAS-PG564/CP05MOAS-PG564_D00004_ingest.csv (hunk 5 above)
Comment (rtravis-whoi) [reply CP05MOAS-PG564/CP05MOAS-PG564_D00004_ingest.csv:8]: good

**Review by rtravis-whoi — APPROVED** [review]
all look good - on RDE.

**Pull request merged**